src/Confluent.Kafka/Consumer.cs

```diff
@@ -645,13 +659,8 @@ public void Assign(IEnumerable<TopicPartition> partitions)
         public void Unassign()
             => kafkaHandle.Assign(null);
 
-        /// <summary>
-        ///     Manually consume message or triggers events.
-        ///
-        ///     Will invoke events for OnPartitionsAssigned/Revoked,
-        ///     OnOffsetCommit, etc. on the calling thread.
-        /// </summary>
-        public bool Consume(out Message message, int millisecondsTimeout)
+
+        internal bool ConsumeImpl(out Message message, int millisecondsTimeout)
```

Comment:
IMO the `out` parameter [should be ordered last](https://msdn.microsoft.com/en-us/library/ms229015(v=vs.110).aspx). It would be also consistent with other BCL APIs.